packages/cli/src/config/config.ts

```diff
@@ -438,6 +438,7 @@ export async function loadCliConfig(
     listExtensions: argv.listExtensions || false,
     extensions: allExtensions,
     blockedMcpServers,
+    vimMode: settings.vimMode,
```

Comment:
revert this as well as it isn't needed as this can just be read off settings directly.